frontend/src/components/EditElectionRoll.tsx

```diff
@@ -6,15 +6,47 @@ import Typography from '@material-ui/core/Typography';
 import Divider from '@material-ui/core/Divider';
 import Container from '@material-ui/core/Container';
 import { Paper, Table, TableBody, TableCell, TableContainer, TableHead, TableRow, Tooltip } from "@material-ui/core";
+import useFetch from "../useFetch";
+import { ElectionRollState } from "../../../domain_model/ElectionRoll";
 
-const EditElectionRoll = ({ roll, onClose }) => {
+const EditElectionRoll = ({ roll, onClose, fetchRolls, id}) => {
     const [updatedRoll, setUpdatedRoll] = useState(roll)
+
+    const approve = useFetch(`/API/Election/${id}/rolls/approve`,'post')
```

Comment:
don't need to address for this diff but:

do we have (or want) an option for a voter to see whether or not they've been approved yet?  (and thus actually counted)?

Reply:
Yes I think so. When the voter goes to the election home page it currently says if they've voted, we can add additional info there.